src/cljs/org/broadinstitute/firecloud_ui/page/workspace/monitor/submission_details.cljs

```diff
@@ -81,6 +81,15 @@
         (merge (select-keys props [:workspace-id :submission-id])
                {:workflow-id (get-in @state [:selected-workflow :id])}))]])})
 
+(defn- getErrorsFromInputResolutions [inputResolutions]
+  (let [errors (filter (fn [m] (contains? m "error")) inputResolutions)
+        keyedErrors (map utils/keywordize-keys errors)]
+       (map (fn [k] (:error k)) keyedErrors)))
+
+(defn- getErrorInputsFromInputResolutions [inputResolutions]
+  (let [errors (filter (fn [m] (contains? m "error")) inputResolutions)
+        keyedErrors (map utils/keywordize-keys errors)]
+       (map (fn [k] (:inputName k)) keyedErrors)))
```

Comment:
You could pass `:error` or `:inputName` as a parameter.  Also, `(fn [k] (:foo k))` reduces to just `:foo`.